Tests/XCUITests/HistoryTests.swift

```diff
@@ -320,6 +320,28 @@ class HistoryTests: BaseTestCase {
         XCTAssertFalse(app.tables.cells.staticTexts[bookOfMozilla["label"]!].exists)
     }
 
+    // Smoke & Functional
+    func testTabHistory() {
+        navigator.nowAt(NewTabScreen)
+        navigator.openURL("example.com")
+        app.windows.otherElements.buttons[AccessibilityIdentifiers.Browser.UrlBar.backButton].press(forDuration: 1)
```

Comment:
If we use `let urlBarBackButton = app.windows.otherElements.buttons[AccessibilityIdentifiers.Browser.UrlBar.backButton]`
and then `urlBarBackButton.press(...)`
the test would look a bit shorter and clearer..